tests/ui/compiletest-self-test/auxiliary/aux_aux_foo.rs

```diff
@@ -0,0 +1,4 @@
+//@ aux-crate: aux_aux_bar=aux_aux_bar.rs
```

Comment:
oh I'm very out of date. I did not get the transition to `aux-crate` and a manual naming scheme. `ui_test` still uses https://github.com/oli-obk/ui_test/blob/main/tests/integrations/basic-fail/tests/actual_tests_bless/auxiliary/nested.rs, but works the same wrt relative naming, so 👍